app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt

```diff
@@ -32,6 +32,18 @@ import com.dessalines.thumbkey.utils.SlideType
 import com.dessalines.thumbkey.utils.SwipeDirection
 import com.dessalines.thumbkey.utils.SwipeNWay
 
+val ABC_KEY_ITEM =
```

Comment:
Good idea. Could you grep for all these, probably with the term `Icons.Outlined.Abc` . I'm sure theres a lot more.